packages/accordion/src/__tests__/Accordion.screenshots.test.tsx

```diff
@@ -43,22 +42,6 @@ describe('Accordion screenshots', () => {
     }
   });
 
-  test('Accordion mouse down on tabs', async () => {
```

Comment:
hover and mouse down no longer have a visual change on the accordion, so these test have no use